### PR TITLE
LVNODCDXI-13

### PR DESCRIPTION
**Pull request by Kindest13**
created db model of user & added post handler on registration path

### CODE REVIEW

File: src/server/controllers/ApiController.js [hunk 1]
```diff
@@ -1,5 +1,26 @@
+import User from '../models/user';
+
 export default class ApiController {
   static index (req, res) {
     res.json({ version: '0.0.1' });
   }
+
+  static postAddUser (req, res, next) {
+    const firstname = req.body.firstname;
```
Comment (tonypntsrn) [inline src/server/controllers/ApiController.js]: This all constants can be distrusted from req.body, like this 
Const { firstname, email, etc.. } = req.body

**Review by tonypntsrn — DISMISSED** [review]

File: src/server/controllers/ApiController.js (hunk 1 above)
Comment (Kindest13) [reply src/server/controllers/ApiController.js]: changed

File: src/server/models/user.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
+const mongoose = require('mongoose');
```
Comment (tonypntsrn) [inline src/server/models/user.js]: Here you could also use es6 imports

**Review by tonypntsrn — DISMISSED** [review]

File: src/server/models/user.js (hunk 2 above)
Comment (Kindest13) [reply src/server/models/user.js]: changed

**Review by tonypntsrn — APPROVED** [review]

**Pull request merged**